### ISSUE TITLE
generated tokens on the localhost are valid on the server !

### ISSUE THREAD

**Issue by Hretic** [tymondesigns/jwt-auth#1147]
im writing  api for a mobile app , the backend website is written in laravel .

so im JWTAuth for authentication like this 

    function  auth()
    {
        try {
            if(!$user = JWTAuth::parseToken()->authenticate())
            {
                return  ('login or register');
            }
        }
        catch (JWTException $e )
        {
            return   ('token is invalid');
        }
        return   $user ;
    }

today i've noticed something  , when i generate an JWT token for a user with id `4` on my localhost api , if i send that same token to the live api on the server  instead of getting error i get the user with id `4` ! 

basically i can login with any user i want just by generating some token on his id on my localhost 

am i missing something ? 

i dont know if this has anything to do with this , but to save time when testing , i dont send tokens as part of POST body and header ... just in the url with `GET` method  like

    site.com/api/user?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9

**Comment by rickshawhobo** [comment]
If you have the same JWT_SECRET on local and live then yes the token is valid for that user id.

**Comment by Hretic** [comment]
thanx that explains it !
